### PR TITLE
Update patches for v87

### PR DESCRIPTION
**Pull request by uazo**
with these should compile, I'm trying

Related: #757

### CODE REVIEW

File: build/patches/Add-history-support-in-incognito-mode.patch [hunk 1]
```diff
@@ -41,31 +40,26 @@ diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/setting
...
   */
-@@ -50,14 +52,16 @@ public class PrivacySettings
+@@ -50,11 +52,13 @@ public class PrivacySettings
      private static final String PREF_ALWAYS_INCOGNITO = "always_incognito";
      public static final String PREF_ALLOW_CUSTOM_TAB_INTENTS = "allow_custom_tab_intents";
      private static final String PREF_PROXY_OPTIONS = "proxy";
--    
-+    public static final String PREF_INCOGNITO_TAB_HISTORY_ENABLED = "incognito_history_enabled";
-+
++public static final String PREF_INCOGNITO_TAB_HISTORY_ENABLED = "incognito_history_enabled";
```
Comment (csagan5) [inline build/patches/Add-history-support-in-incognito-mode.patch:47]:
```suggestion
+    public static final String PREF_INCOGNITO_TAB_HISTORY_ENABLED = "incognito_history_enabled";
```

File: build/patches/Add-history-support-in-incognito-mode.patch [hunk 2]
```diff
@@ -117,7 +111,7 @@ diff --git a/chrome/browser/history/history_tab_helper.cc b/chrome/browser/histo
 +#if defined(OS_ANDROID)
 +  if(profile->GetOriginalProfile()->GetPrefs()->GetBoolean(prefs::kIncognitoTabHistoryEnabled)) {
```
Comment (csagan5) [inline build/patches/Add-history-support-in-incognito-mode.patch:112]:
```suggestion
+        if(profile->GetOriginalProfile()->GetPrefs()->GetBoolean(prefs::kIncognitoTabHistoryEnabled)) {
```

File: build/patches/Add-history-support-in-incognito-mode.patch [hunk 3]
```diff
@@ -117,7 +111,7 @@ diff --git a/chrome/browser/history/history_tab_helper.cc b/chrome/browser/histo
...
 +    return HistoryServiceFactory::GetForProfile(profile, ServiceAccessType::IMPLICIT_ACCESS);
```
Comment (csagan5) [inline build/patches/Add-history-support-in-incognito-mode.patch:113]:
```suggestion
+        return HistoryServiceFactory::GetForProfile(profile, ServiceAccessType::IMPLICIT_ACCESS);
```

File: build/patches/Add-history-support-in-incognito-mode.patch [hunk 4]
```diff
@@ -117,7 +111,7 @@ diff --git a/chrome/browser/history/history_tab_helper.cc b/chrome/browser/histo
...
-+  } 
++  }
```
Comment (csagan5) [inline build/patches/Add-history-support-in-incognito-mode.patch:114]:
```suggestion
+        }
```

File: build/patches/Show-site-settings-for-cookies.patch [hunk 5]
```diff
@@ -58,51 +58,28 @@ diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/c
...
+-        // Not possible to embargo JAVASCRIPT.
 +        // Not possible to embargo COOKIES.
          setUpListPreference(preference, currentValue, false /* isEmbargoed */);
      }
  
-@@ -932,22 +941,10 @@ public class SingleWebsiteSettings extends SiteSettingsPreferenceFragment
-             setUpListPreference(preference, null, false);
-             return;
-         }
--        // If the ad blocker is activated, then this site will have ads blocked unless there is an
```
Comment (csagan5) [inline build/patches/Show-site-settings-for-cookies.patch:87]: This part is still needed, I will try to restore it.

**Comment by csagan5** [comment]
Thanks for this PR; I will restore the part about the ads and check if everything works.

**Comment by uazo** [comment]
God forbid! thanks for the experience, I learned a lot (especially that I lack some tools :)

**Pull request merged**